examples/web-show/deploy.sh

```diff
@@ -63,6 +63,10 @@ fi
 
 TARGET_IP=$(kubectl get pod -n kube-system -o wide| grep kube-controller | head -n 1 | awk '{print $6}')
 
+if [ -z "${TARGET_IP}" ]; then
+   TARGET_IP=$(kubectl get pod -n kube-system -o wide | grep Running | head -n 1 | awk '{print $6}')
+fi
+
```

Comment:
Maybe it would be better if we deployed another nginx as the target server, depending on the platform environment. This doesn't work on Ali Cloud K8S and openshift

Reply:
Yes, I will close this pr.